src/database/db_build.sql

```diff
@@ -0,0 +1,79 @@
yqMT1HmOTh3/vK608Ys+UbMfpa1K0BVL6hqPjcNVyxWKJIa27j6byeSlUmhgDRrvPEpSL2BqNkN0M/UK3pOAboqPF4cPF9dQeEb+RXJ2OfTEOBcBod/7quv0i5fbl0lxLKdzvny3rK1MzqYjqMbssiNo6xvgJ+cPq4uoJBa3nvVrgcCKYAhdZjJPLjlncr49JOW9Road1vBTDW3tvyVbiKobKjYPNQSWQS6x2QJgc+Cm47XM1rUqSJEqmre8O2Fe4OiSSSIBCpMwGzUA5ylj7GfmbVudWsh0Yr0G1v+oaCxzdkFwBDXFwgnhwnmm56+8Kkr3a4b4Mduo9F3w9M3I9rpsAADQABoAABHIBFYzo50pZTw7P4ja2QAA8DaDW7tqLwOK2FCs2o0PY4Oa4SHNIII5ELpZpmdWlPJTAikClCUChKDOlJNSQNLVJFAqkggSimFIxRQCSQIrlVYHCCLLoSmlFOKDOcFst2wSYMGT+E/v6qtoiSOa1VemHAtOhEHsKyJBY5zDq0+PArhyY/TZw579umwWPe47wAOzf5lcXZlfR0DeQY8Va4eq17RKIoAgjdw+K5S/bRZ9IdPHMi72jvTH4yiRBeTf8LT6qL/AGexlTaDRYz9clpMBmdOA0Ur7QNo+p1VzX2MsMv1NqFhpgwyk97r22T9cEalHFPmKbWACYdrF/l5rV4rNSSC1gni8yY36fNVGZYhzgXON7gBthHPiquk44Z39aYjMqNd8AVYB1hoHcFZ9G8ubTBjnJNyTvk703FVPE2A4KwwFPZbG9K5eFXGS+PK0p1QGlZDM633o4K9r14BWWxb+sTwU4TyjkvjStzWttPPJQaRkylUcXHtXVlOAtHpm91bZM0GgJFgXMPcY9FBwmaV8sr/AHZ2qDzPsz7pG+PyuHHxVxlNCMKXfmqujyHwUPP8MH0Sd7b+C1a3GO3VeiZHnlHFs26Tp/M02c08HD46KzBXzzgcbUovFSk9zHDQtMdx4jkV6P0c+0Nj4ZigGO/1B7h/UPwenYuNxsXLtvymEJtKqHAOaQQbggyCORTiVIBJKUEGt0Ekk0kmFPKagxQKKRQDECnIJBzcFR9IcvLh7Vg67Rcfmb8wr5wXMtU2Lxy0xOExe8KypYhLpHlYaDXpCDPXbuIP4hwMwqrCYkftwXDPDTdx8m4vK1HbEjVQXuLTpHmpuCrgqVVaCue9O3+KcYqsdHAf+K51ab3C8uPMwFeMwjY0Tv4dsaK9l5UWFy/ee5S61LZCkuMWVfmGJhpJ7VHm07qRU43FahZvNMTaBvTsXjNpx4KEGF7loxx0yZZdhwVCbqxqYfqwpOEwsBWmXZf7WoxkWJv+kXPy7055uj11xtrvVwvs8NQZvIDj2u6x9VVZlShjuw+i1GcjaqgDQWVTnFKKbz/KfRehp5e3lDwgE+qEwLk6LrIOk+IwcikQWG5Y8EtneRBGyexa7B/aYP8ANw5HOm+f9rgPVecIgJXGU9vZ8v6ZYKr/AJwYeFX7vwJse4oLx3ZSU9D7PplJJJQAKQRQQBQKSSACSa8xcmBzWX6QdOsNhpa0+1qflYRA/U7QdiNG1JWUz7pzhcOSxp9q8fhZEA8C7ReeZ903xWJBYXCnTP4WSJHAu1KzbQrmH2ns1+adPsVWlrdmkwzZo2iRzcfkrb2Ti1pI2XFrXRyLQ4eq88+K9Q6SYwMxjaBED2LNk6TE2B329EuTCddx14c/7aqPl2ZbD9l9p3rSsrTvWZq0A5MpV6lO2o4H4LJcZfTbjncfFbL24hcG4sAmSsm/OH8PNRqua1Du80v46r+WNNmOYsaLG6yWa5mXDZB11+SiV6rnanwUb2ZXTHCRyyzuTiGyrfA4WLrlg8NdXeGohLPI8MD6NALUdHsHsU3Vjq4bLf0jXxPoqbA4U1Hhg3mOwbz4Stlj2BjAxtg0ABdvjYbvauXy8+s6xmaresSqvpA+KFQ/yu9FcV7LNdKqsUHniNnxIHxW157zWsFyCkVFxK5LIJ0IBPCRi02STW69qSZPppJJJcVgkmVqrWAuc4NaNSTAHesL0i+0elTlmGHtHfnNmDs3uRIG4xWKZTaX1HtY0alxACw+ffaXRZLcM32jvzu6rB2b3LzXNs4r4l+3WqOcdwJ6o/S3QKArmJbWmbdIsTiTNWq4j8oOy0djRZVaKSrRE1qe5IFNcmRL1fpzgfb4LDY6n79NjCSPykAnwPxXlFPVe2/Z3WbicuFJ19japEchp5EJ/ob1WZyzFCrTDhroRwKkPAVOMK7A4w0nf3dT3TzBt33jwV/UpzcLDnOuT0uO9sdq+phwVEqYfgFZQmlvLyU7PrFScOn08NKsPZDh5J+yi05ijUqICn0G6ei5U6ZJt4q5yzAlzwwb9TwG/sU63dL3JNrzovl+w01Xamw7Pr0T81rXVrWhjQ0WAELGZ1jSXbDdSvTwx64yPI5M7nla6VesDHasZ0yq/c7PF7R6n4Le0cNs05OsLzvpwNltIfmc93gAB6qr6RGPe1cHBSSmubK5rcQnBBwA1QAJ5Dz/AGSAO5JLo1sJJh9MLKdLOm1HCAsZFSt+UGzebyPRW/SioW4Ss5pIIpuggwRbcQvn95JMlc5NqqzzrpDiMUZrVCRNmizR2NVUUSg5dNJCE6EAnIAFBIooBpSJScgUA+kvRPsfzPYxFWgTao0Pb+ptneRHgvPKS1H2ef46n3qoVekdPMjFZhj3h1mng4fMLI9H802x7OpaoyxnlZeq5mOp3j1C8ZzkRjrWkjS25cefCWbafj52VpqtHeP3TQ3mjhT1URoViehpyNLmnGmOHefgnbimVTbvRonfDU5NvK8rbZJl/s27Th1j5DgoWR0W+zY7ZG1AvAm5vdaB+i2cPDMf7Vg+Rz3L+kUuc4nZBWMy/r1i48VoOkhsVR5H7xWpjaWp7sLzP7SLVKLf5HHxdH/FemVNF5f9pv8Ai2/9lv8A9PRfRz2yJQRKS5Lc3MlKeKeU1yASSazVJAf/2Q=='
```

Comment:
well in real life its going to be the S3 url, so maybe to prepare better for the future, have a url instead of base64

Reply:
done